cli_tutorials/cli_charge_molecules.md

```diff
@@ -0,0 +1,103 @@
+# Generating Partial Charges with the OpenFE CLI
+
+This tutorial will show you how to use the OpenFE CLI to generate and store partial charges for a series of ligands 
```

Comment:
The order of the intro feels a little bit off. I think I would lead with "you need consistent charges". 

Then say "by default the behaviour is X".

Then say "here we present tooling to allow you to do this ahead of time, reducing overheads and further improving reproducibility".